### PR TITLE
docs: Add BurdaForward to "In the Wild" and merge "News" and "Entertainment" to "Media & Entertainment"

### PR DESCRIPTION
**Pull request by mattitoo**
### SUMMARY

 Add BurdaForward to "In the Wild" and merge "News" and "Entertainment" to "Media & Entertainment"

### TESTING INSTRUCTIONS

Check the page if link for BurdaForward works

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: RESOURCES/INTHEWILD.md [hunk 1]
```diff
@@ -125,6 +125,8 @@ Join our growing community!
 - [TME QQMUSIC/WESING](https://www.tencentmusic.com/)[@shenyuanli,@marklaw]
 - [Xite](https://xite.com/) [@shashankkoppar]
 - [Zaihang](http://www.zaih.com/)
+- [Prensa Iberica](https://www.prensaiberica.es/) [@zamar-roura]
+- [BurdaForward](https://www.burda-forward.de/en/)
```
Comment (villebro) [inline RESOURCES/INTHEWILD.md:129]: Please place the entries in ABC order

**Review by villebro — CHANGES_REQUESTED** [review]
Thanks for the addition - one minor comment about ordering, other than that LGTM

**Review by villebro — APPROVED** [review]
LGTM

**Comment by rusackas** [comment]
Feel free to update the new [yaml file](https://github.com/apache/superset/edit/master/RESOURCES/INTHEWILD.yaml) if you want to upload a logo or add additional github handles!

**Pull request merged**